ansible_collections/arista/avd/examples/single-dc-l3ls/switch-basic-configurations/dc1-leaf1a-basic-configuration.txt

```diff
@@ -0,0 +1,41 @@
+! ansible-avd-examples/single-dc-l3ls/switch-basic-configurations/dc1-leaf1a-basic-configuration.txt
```

Comment:
Could we find a more elegant way to get started instead of all this config?

Reply:
Open to suggestions, however I don't see how we can make it *much* simpler.
Only alternative I can think of describing ZTP without CVP, which I doubt will be simpler, faster or more elegant.